File: app/src/main/java/com/yoesuv/infomalangbatu/fragment/ChildFragmentInfo.java
package com.yoesuv.infomalangbatu.fragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.yoesuv.infomalangbatu.BuildConfig;
import com.yoesuv.infomalangbatu.R;
import com.yoesuv.infomalangbatu.utils.GravityLightTextView;

public class ChildFragmentInfo extends Fragment {

    public static ChildFragmentInfo getInstance(){
        return new ChildFragmentInfo();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        View v = inflater.inflate(R.layout.fragment_info, container, false);

        GravityLightTextView textVersion = v.findViewById(R.id.textView_version_app);
        String appVersion = "Versi "+ BuildConfig.VERSION_NAME;
        textVersion.setText(appVersion);

        return v;
    }
}

File: app/src/main/java/com/yoesuv/infomalangbatu/menu/other/views/ChildFragmentInfo.kt
package com.yoesuv.infomalangbatu.menu.other.views

import android.content.Intent
import androidx.databinding.DataBindingUtil
import android.net.Uri
import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.yoesuv.infomalangbatu.BuildConfig
import com.yoesuv.infomalangbatu.R
import com.yoesuv.infomalangbatu.databinding.ChildFragmentInfoBinding

class ChildFragmentInfo : Fragment() {

    companion object {
        fun getInstance(): Fragment {
            return ChildFragmentInfo()
        }
    }

    private var binding: ChildFragmentInfoBinding? = null

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        if (binding == null) {
            binding = DataBindingUtil.inflate(inflater, R.layout.child_fragment_info, container, false)
        }
        return binding?.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        binding?.textViewVersion?.text = getString(R.string.info_app_version, BuildConfig.VERSION_NAME)
        binding?.textViewGithub?.setOnClickListener {
            val intent = Intent(Intent.ACTION_VIEW, Uri.parse(context?.getString(R.string.link_github)))
            context?.startActivity(intent)
        }
    }

}